distribution/pom.xml

```diff
@@ -85,8 +87,7 @@
                 <configuration>
                     <archive>
                         <manifest>
-                            <addClasspath>true</addClasspath>
```

Comment:
Removed the classpath here as it doesn't work correctly either way with -jar or -cp. With -cp it is useless because ignored and with -jar it works but jpeg2000 or other optional libraries cannot be loaded.